src/platform-includes/capture-error/javascript.sveltekit.mdx

```diff
@@ -0,0 +1,11 @@
+You can pass an `Error` object to `captureException()` to get it captured as event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stacktrace.
```

Comment:
```suggestion
You can pass an `Error` object to `captureException()` to get it captured as event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stack trace.
```

I know this is just copied from existing docs, but stacktrace isn't correct, so let's fix it here at least